### ISSUE TITLE
wahooMapsCreator-v2.0.0-Windows.zip: Location of `tooling_win` incorrect

### ISSUE THREAD

**Issue by q74cll** [treee111/wahooMapsCreator#121]
# Code to reproduce
~~~ps1
Invoke-WebRequest -URI 'https://github.com/treee111/wahooMapsCreator/releases/download/v2.0.0/wahooMapsCreator-v2.0.0-Windows.zip' -OutFile "$DownloadFolder\wahooMapsCreator-current.zip"
$WahooMapsCreatorHome="$DownloadFolder\wahooMapsCreator-current"
New-Item -Type directory -Force -Path $WahooMapsCreatorHome
Expand-Archive -Path "$DownloadFolder\wahooMapsCreator-current.zip" -DestinationPath $WahooMapsCreatorHome

$GdalEnvironment="C:\Program Files\Anaconda3\envs\gdal-user"
$DownloadFolder="$HOME\Desktop\wahooMapsCreator"
$WahooMapsCreatorHome="$DownloadFolder\wahooMapsCreator-current"
cd $WahooMapsCreatorHome
Remove-Item -Recurse -Force $GdalEnvironment
powershell -ExecutionPolicy ByPass -NoExit -Command "& 'C:\Program Files\Anaconda3\shell\condabin\conda-hook.ps1' ; conda activate 'C:\Program Files\Anaconda3'"
conda env create -f .\conda_env\gdal-user.yml

$DownloadFolder="$HOME\Desktop\wahooMapsCreator"
$WahooMapsCreatorHome="$DownloadFolder\wahooMapsCreator-current"

cd $WahooMapsCreatorHome
conda activate gdal-user
python -m wahoomc cli -co malta -fp -fd -tag tag-wahoo.xml
~~~

Output:
~~~
INFO:--------------------------------------------------------------------------------
INFO:# Input country: malta.
INFO:--------------------------------------------------------------------------------
INFO:# Determine involved/border countries
INFO:+ Involved country: malta
INFO:--------------------------------------------------------------------------------
INFO:# check C:\Users\gercri\wahooMapsCreatorData\_download\land-polygons-split-4326\land_polygons.shp file
INFO:+ Deleting old C:\Users\gercri\wahooMapsCreatorData\_download\land-polygons-split-4326\land_polygons.shp file
INFO:+ C:\Users\gercri\wahooMapsCreatorData\_download\land-polygons-split-4326\land_polygons.shp file needs to be downloaded
INFO:# Need to download land polygons file
INFO:--------------------------------------------------------------------------------
INFO:+ Downloading land_polygons.shp file
INFO:+ Downloaded: C:\Users\gercri\wahooMapsCreatorData\_download\land-polygons-split-4326\land_polygons.shp
INFO:+ check land_polygons.shp file: OK
INFO:--------------------------------------------------------------------------------
INFO:# check countries .osm.pbf files
INFO:+ Checking for old maps and remove them
INFO:+ mapfile for malta: deleted. Input: malta.
INFO:--------------------------------------------------------------------------------
INFO:+ Downloading malta-latest.osm.pbf file
INFO:+ Downloaded: C:\Users\gercri\wahooMapsCreatorData\_download\maps\malta-latest.osm.pbf
INFO:+ Check countries .osm.pbf files: OK
INFO:--------------------------------------------------------------------------------
INFO:# Filter tags from country osm.pbf files
INFO:! cmd.append is ['C:\\Users\\gercri\\Desktop\\wahooMapsCreator\\wahooMapsCreator-current\\wahoomc\\tooling_win\\osmconvert64-0.8.8p', '-v', '--hash-memory=2500', '--complete-ways', '--complete-multipolygons', '--complete-boundaries', '--drop-author', '--drop-version', 'C:\\Users\\gercri\\wahooMapsCreatorData\\_download\\maps\\malta-latest.osm.pbf', '-o=C:\\Users\\gercri\\wahooMapsCreatorData\\_tiles\\outFile-malta.o5m']:
Traceback (most recent call last):
  File "C:\Program Files\Anaconda3\envs\gdal-user\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Program Files\Anaconda3\envs\gdal-user\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\gercri\Desktop\wahooMapsCreator\wahooMapsCreator-current\wahoomc\__main__.py", line 7, in <module>
    main.run()
  File "C:\Users\gercri\Desktop\wahooMapsCreator\wahooMapsCreator-current\wahoomc\main.py", line 47, in run
    o_osm_maps.filter_tags_from_country_osm_pbf_files()
  File "C:\Users\gercri\Desktop\wahooMapsCreator\wahooMapsCreator-current\wahoomc\osm_maps_functions.py", line 269, in filter_tags_from_country_osm_pbf_files
    cmd, '! Error in OSMConvert with country: {key}')
  File "C:\Users\gercri\Desktop\wahooMapsCreator\wahooMapsCreator-current\wahoomc\osm_maps_functions.py", line 79, in run_subprocess_and_log_output
    with subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT) as process:
  File "C:\Program Files\Anaconda3\envs\gdal-user\lib\subprocess.py", line 800, in __init__
    restore_signals, start_new_session)
  File "C:\Program Files\Anaconda3\envs\gdal-user\lib\subprocess.py", line 1207, in _execute_child
    startupinfo)
FileNotFoundError: [WinError 2] The system cannot find the file specified
~~~

# Solution
- `tooling_win` must reside inside `wahoomc`

~~~ps1
$DownloadFolder="$HOME\Desktop\wahooMapsCreator"
$WahooMapsCreatorHome="$DownloadFolder\wahooMapsCreator-current"

Move-Item $WahooMapsCreatorHome/tooling_win $WahooMapsCreatorHome/wahoomc/tooling_win
~~~

**Comment by treee111** [comment]
thanks for creating the bug!
Fix is out via v2.0.1 both .zip and via PyPI :-)

Would be nice if you could check if it is working for you now, thanks! 👍

**Comment by q74cll** [comment]
# Test with v2.0.1

**Output:**
~~~
INFO:--------------------------------------------------------------------------------
INFO:# Input country: malta.
INFO:--------------------------------------------------------------------------------
INFO:# Determine involved/border countries
INFO:+ Involved country: malta
INFO:--------------------------------------------------------------------------------
INFO:# check C:\Users\gercri\wahooMapsCreatorData\_download\land-polygons-split-4326\land_polygons.shp file
INFO:+ Deleting old C:\Users\gercri\wahooMapsCreatorData\_download\land-polygons-split-4326\land_polygons.shp file
INFO:+ C:\Users\gercri\wahooMapsCreatorData\_download\land-polygons-split-4326\land_polygons.shp file needs to be downloaded
INFO:# Need to download land polygons file
INFO:--------------------------------------------------------------------------------
INFO:+ Downloading land_polygons.shp file
INFO:+ Downloaded: C:\Users\gercri\wahooMapsCreatorData\_download\land-polygons-split-4326\land_polygons.shp
INFO:+ check land_polygons.shp file: OK
INFO:--------------------------------------------------------------------------------
INFO:# check countries .osm.pbf files
INFO:+ Checking for old maps and remove them
INFO:+ mapfile for malta: deleted. Input: malta.
INFO:--------------------------------------------------------------------------------
INFO:+ Downloading malta-latest.osm.pbf file
INFO:+ Downloaded: C:\Users\gercri\wahooMapsCreatorData\_download\maps\malta-latest.osm.pbf
INFO:+ Check countries .osm.pbf files: OK
INFO:--------------------------------------------------------------------------------
INFO:# Filter tags from country osm.pbf files
INFO:+ Converting map of malta to o5m format
INFO:+ Filtering unwanted map objects out of map of malta
INFO:+ Filter tags from country osm.pbf files: OK
INFO:--------------------------------------------------------------------------------
INFO:# Generate land
INFO:+ Generate land 1 of 1 for Coordinates: 138,100
INFO:+ Generate land: OK
INFO:--------------------------------------------------------------------------------
INFO:# Generate sea
INFO:+ Generate sea 1 of 1 for Coordinates: 138,100
INFO:+ Generate sea: OK
INFO:--------------------------------------------------------------------------------
INFO:# Split filtered country files to tiles
INFO:+ Splitting tile 1 of 1 for Coordinates: 138,100 from map of malta
INFO:+ Split filtered country files to tiles: OK
INFO:--------------------------------------------------------------------------------
INFO:# Merge splitted tiles with land an sea
INFO:+ Merging tiles for tile 1 of 1 for Coordinates: 138,100
INFO:--------------------------------------------------------------------------------
INFO:# Sorting land* osm files
INFO:+ Sorting land* osm files: OK
INFO:+ Merge splitted tiles with land an sea: OK
INFO:--------------------------------------------------------------------------------
INFO:# Creating .map files
INFO:+ Creating map file for tile 1 of 1 for Coordinates: 138,100
INFO:+ Creating .map files: OK
INFO:--------------------------------------------------------------------------------
INFO:# Create: .map.lzma files
INFO:+ Country: malta
INFO:+ Copying .map.lzma tiles to output folders
INFO:+ Create .map.lzma files: OK
~~~

# Result

**TEST OK**
